packages/base/Table/src/TableRow/TableRow.tsx

```diff
@@ -36,30 +30,26 @@ export const TableRow = forwardRef<HTMLTableRowElement, Props>(
       onClick,
       ...rest
     } = props
-    const {
-      stripeEven: stripeEvenClass,
-      bordered: borderedClass,
-      ...muiClasses
-    } = useStyles()
+
     const { variant } = useContext(TableContext)
     const isBordered = variant === 'bordered' || variant === 'striped'
 
     return (
-      <MUITableRow
+      <tr
         {...rest}
         ref={ref}
-        classes={muiClasses}
-        className={cx(className, {
-          [stripeEvenClass]: stripeEven,
-          [borderedClass]: isBordered,
-        })}
+        className={twMerge(
+          isBordered && 'border-0 border-solid border-b border-gray-200',
+          stripeEven && 'bg-gray-100',
+          hover && 'hover:bg-blue-100',
```

Comment:
```suggestion
          'hover:bg-blue-100',
          hover && 'bg-blue-100'
```

`hover:` should always be assign, but if "hover" prop is passed we must force set "bg-blue-100"

Reply:
The previous statement is wrong, however, as I can see, we anyway should specify "hover:" style for the following states: "selected, hover"

Reply:
I'm not sure that I understand your comment correctly, but I think we don't need to apply the `hover` style for the `selected` state, as in this case, the row will lose its background color when selected and hovered.